code/src/WijmoProvider/Features/ConditionalFormat.ts

```diff
@@ -13,6 +13,24 @@ namespace WijmoProvider.Feature {
     function Evaluate(operator: Rules, comparedValue: any, cellValue: any) {
         // in case we are comparing dates
         if (typeof cellValue.getMonth === 'function') {
+            // Whenever we have null dates coming from OS, it has a different timezone than today's
+            // this is the way JS handles dates before 1911: "historical timezone offsets are applied, prior to about 1900 most were not even hour or half hour offsets."
+            // so we must ensure that our compared value (OS Null date) has this GMT as well.
+            const comparedDate = new Date(comparedValue);
+            const comparedYear = comparedDate.getUTCFullYear();
+            if (comparedYear <= 1911) {
```

Comment:
why 1911?

Reply:
It seems that prior to 1911 time zones were different from today. Javascript respects these rules, so whenever we compared dates from 1900 (null date) a weird timezone was being applied, making it impossible to compare the values and apply conditional format.